File: class10/java/AP1004-MatrizBasica.java
public class MatrizBasica {
    public static void main(String[] args) {
        int[][] m = {{1, 2, 3},
                     {4, 5, 6}};
        System.out.println("m[0][2] = " + m[0][2]);   // 3
        System.out.println("m[1][0] = " + m[1][0]);   // 4
        System.out.println("Matriz completa:");
        for (int i = 0; i < m.length; i++) {
            for (int j = 0; j < m[i].length; j++) {
                System.out.print(m[i][j] + " ");
            }
            System.out.println();
        }
    }
}

File: class10/python/AP1004-MatrizBasica.py
# Matriz (variavel indexada bidimensional): declaracao e acesso m[i][j]
m = [[1, 2, 3],
     [4, 5, 6]]
print("m[0][2] =", m[0][2])   # 3
print("m[1][0] =", m[1][0])   # 4
print("Matriz completa:")
for i in range(len(m)):
    for j in range(len(m[i])):
        print(m[i][j], end=" ")
    print()
